lib/engine/game/g_18_eus/step/buy_sell_par_shares.rb

```diff
@@ -4,12 +4,14 @@
 require_relative 'parrer'
 require_relative 'bidbox_auction'
 require_relative 'loan_taker'
+require_relative 'game_button_provider'
 
 module Engine
   module Game
     module G18EUS
       module Step
         class BuySellParShares < Engine::Step::BuySellParShares
+          include GameButtonProvider
```

Comment:
I'm not comfortable with this being an include. The modules that use it should inherit from it instead.